_data/versions.yml

```diff
@@ -1,4 +1,55 @@
 v2.6:
+- title: v2.6.3
+  note: |
+    28 November 2017
+    Calico v2.6.3 addresses several outstanding common vulnerabilities and exposures (CVE).
```

Comment:
Looks like there are bullet points under this title, which isn't quite right - the things under this aren't CVEs

Reply:
I would say I agree... @bcreane this sentence is pretty broad, any further details on these? Either way, we could make it it's own bullet